src/style.css

```diff
@@ -54,12 +53,20 @@
   font-weight: bold;
 }
 
-.box-sm:focus,
 .box-sm:hover {
   background-size: 11px 11px;
   background-position: 12px center;
 }
 
+.box-sm:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(./images/Search.svg);
```

Comment:
Your `background-image` is overriding `background` property.
https://developer.mozilla.org/en-US/docs/Web/CSS/background